libs/vec/native/src/vec/c/aarch64/vec.c

```diff
@@ -121,7 +121,7 @@ static inline int32_t sqr7u_inner(int8_t *a, int8_t *b, size_t dims) {
 EXPORT int32_t sqr7u(int8_t* a, int8_t* b, size_t dims) {
     int32_t res = 0;
     int i = 0;
-    if (i > SQR7U_STRIDE_BYTES_LEN) {
+    if (dims > SQR7U_STRIDE_BYTES_LEN) {
```

Comment:
Oops, thank you.

Reply:
No prob! It drove me mad yesterday because I was seeing it running so much slower :) 
I understood what was going on only after disassembling the function and seeing no inline vector code nor function call...
Thanks for updating the title!